packages/core/crates/core-mixer/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "core-mixer"
+version = "0.1.0"
+authors = ["Tibor Csóka"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook", "wasm"]
+wasm = []
```

Comment:
Please use the feature to automatically enable the wasm dependencies, e.g. `wasm = ["dep:wasm-bindgen", "dep:serde-wasm-bindgen", "dep:js-sys", "dep:wasm-bindgen-futures" ]
`

Reply:
`error: failed to parse manifest at `hoprnet/packages/core/crates/core-mixer/Cargo.toml``